### DISCUSSION TITLE
Ability to apply the changed pitch in the main menu

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#13161] [Ideas]
**Describe the new feature:** 
Some songs go perfectly with DT/HT/DC/NC mods. When these mods are activated in the map selection section, the pitch changes, but when you exit to the main menu, the pitch is lost. Can you add the ability not to change the pitch of the active mod?

It can be a toggle checkbox in the settings.

**Comment by smoogipoo** [comment]
This was intentional - all mods are removed when exiting to main menu. I'm not sure this will change.
